### PR TITLE
[Themes]: Update RTE

### PR DESCRIPTION
**Pull request by siarhei-epam**
Add variables to RTE, remove SkinContext from UuiContexts, add neutral color to Button in uui package, move RichTextView, VirtualList, ScrollBars, Checkbox from skins to uui package, reverse neutral colors for "loveship dark" theme, update demos & examples

### CODE REVIEW

File: uui-editor/src/implementation/ColorBar.tsx [hunk 1]
```diff
@@ -16,28 +14,28 @@ type IColorBar = {
 
 export function ColorBar({ updateColor, clearColor, value }: IColorBar) {
     return (
-        <FlexRow rawProps={ { style: { background: '#303240' } } }>
+        <FlexRow rawProps={ { style: { border: '1px solid var(--uui-divider-light)' } } }>
```
Comment (AlekseyManetov) [inline uui-editor/src/implementation/ColorBar.tsx:17]: let's move it to the css class

File: uui-editor/src/implementation/HeaderBar.tsx [hunk 2]
```diff
@@ -44,10 +43,9 @@ export class HeaderBar extends React.Component<HeaderBarProps> {
         const block = getBlockAbove(this.props.editor);
 
         return (
-            <FlexRow rawProps={ { style: { height: 42, background: '#303240' } } }>
+            <FlexRow rawProps={ { style: { border: '1px solid var(--uui-divider-light)' } } }>
```
Comment (AlekseyManetov) [inline uui-editor/src/implementation/HeaderBar.tsx:46]: let's move it to the css class

File: uui-editor/src/implementation/NoteBar.tsx [hunk 3]
```diff
@@ -39,7 +36,7 @@ export function NoteBar({ editor, type }: NoteBarProps) {
     };
 
     return (
-        <FlexRow rawProps={ { style: { background: '#303240' } } }>
+        <FlexRow rawProps={ { style: { border: '1px solid var(--uui-divider-light)' } } }>
```
Comment (AlekseyManetov) [inline uui-editor/src/implementation/NoteBar.tsx:39]: let's move it to the css class

File: uui-editor/src/plugins/tablePlugin/TableCell.module.scss [hunk 4]
```diff
@@ -21,25 +17,24 @@
 }
 
 .border-bottom::before {
-    border-bottom: 1px solid #D1D5DB;
+    border-bottom: 1px solid var(--uui-control-border);
```
Comment (AlekseyManetov) [inline uui-editor/src/plugins/tablePlugin/TableCell.module.scss:20]: May be better to use here table variables?

**Review by AlekseyManetov — CHANGES_REQUESTED** [review]

**Review by vik753 — APPROVED** [review]

File: app/src/docs/_examples/mainMenu/Responsive.example.tsx [hunk 5]
```diff
@@ -1,5 +1,22 @@
 import React, { ReactNode, useState } from 'react';
-import { Slider, BurgerButton, GlobalMenu, MainMenu, MainMenuAvatar, MainMenuButton, MainMenuIcon, FlexSpacer, FlexCell, MainMenuDropdown, BurgerSearch, DropdownMenuBody, DropdownMenuButton, DropdownMenuSplitter, Burger } from '@epam/uui';
+import {
```
Comment (AlekseyManetov) [inline app/src/docs/_examples/mainMenu/Responsive.example.tsx:2]: Fix imports layout

**Review by AlekseyManetov — CHANGES_REQUESTED** [review]

File: uui-editor/src/SlateEditor.tsx [hunk 6]
```diff
@@ -83,21 +84,21 @@ function Editor(props: PlateEditorProps) {
...
-                props.fontSize === '16' ? css.typography16 : css.typography14,
+                props.fontSize === '16' ? 'uui-typography-size-16' : 'uui-typography-size-14',
             ) }
             style={ { minHeight: props.minHeight || 350 } }
             { ...props.rawProps }
         >
             { props.scrollbars
                 ? (
-                    <ScrollBars cx={ css.scrollbars } style={ { width: '100%' } }>
+                    <ScrollBars style={ { width: '100%' } }>
```
Comment (AlekseyManetov) [inline uui-editor/src/SlateEditor.tsx:101]: move to the css

**Review by AlekseyManetov — CHANGES_REQUESTED** [review]

**Pull request merged**